tests/e2e/molecule/README.MD

```diff
@@ -0,0 +1,38 @@
+# cloud.vmware_ops Molecule E2E Testing
+
+This repository contains Molecule test scenarios for the `cloud.vmware_ops` Ansible collection. Each scenario represents a feature in the collection, allowing you to test individual features in isolation.
+Feel free to contribute, add new scenarios, or enhance existing ones to ensure comprehensive testing of the Ansible collection.
+
+## Prerequisites
+
+Before running Molecule scenarios, ensure that the following tools are installed on your system:
+
+- [Ansible](https://www.ansible.com/)
+- [Python 3](https://www.python.org/)
+- [Molecule](https://molecule.readthedocs.io/)
+- [ansible-junit](https://github.com/hspaans/ansible-junit) (for JUnit XML output)
+- [pyvmomi](https://github.com/vmware/pyvmomi) (required for vSphere testing)
+
+Additionally, create a password file for the vSphere credentials (vault password file). The password file should contain the vault password for the `vSphere creds` vault. The vault file should include the following variables:
+
+```yaml
+# vault.yml
+vCenter_hostname: "your_vsphere_hostname"
```

Comment:
Is there any reason to use capital letter in the name? Those won't pass ansible-lint by default. I think it's better to not use those, unless there is a really strong reason to do so.

Reply:
thanks, updated to use only lowercase